README.md

```diff
@@ -1 +1,2 @@
-# Datatrail-analysis
\ No newline at end of file
+
+![first plot](cloud/project/plots)
```

Comment:
```suggestion
![first plot](plots/plot1.png)
![second plot](plots/plot2.png)
```

Markdown is relative file path. Take a read of this: https://datatrail-jhu.github.io/DataTrail/managing-files.html#file-system-example-and-vocabulary

Reply:
You were very close!